recipes/bioconductor-mspurity/meta.yaml

```diff
@@ -37,9 +36,14 @@ requirements:
     - r-rsqlite
     - r-stringr
     - r-uuid
+    - r-optparse    
+    - r-rpostgres
+    - r-rmysql
+    - 'bioconductor-xcms >=3.8.0,<3.9.0'
+    - 'bioconductor-camera >=1.42.0,<1.42.1'
+    - 'bioconductor-mspuritydata >=1.14.0,<1.15.0'
```

Comment:
As a policy we don't install suggested packages unless they're really needed for normal use. Are these really needed for most normal use of the package? If so, can you suggest to the bioconductor package maintainer that they be moved to dependencies in the next release?

Reply:
I am the maintainer... :) so I can add for the next release. They are needed for the functionality required for our Galaxy workflows - the majority of the use cases of the package does not require the additional packages though. However, I can add these packages to the standard "import" section for the next release of msPurity

Reply:
@dpryan79, just checking if the pull request is OK as it is?

Reply:
Why not just add these as dependencies in the galaxy wrapper then?

Reply:
I just think it is cleaner to have just one dependency in Galaxy when possible (i.e. so the version of the tool can follow the IUC versioning - e.g. 1.12.0+galaxy1 and the conda environment is not mulled)

I can update the release in bioconductor to add the required packages from the "suggest" section to the "import" section? 

And I guess the new version (1.12.1) will then automatically build for bioconda will all the correct dependencies?

Reply:
The version of the tool wrapper can still follow the version of the tool, even with a mulled container (this is the case for the deepTools wrapper that I maintain, since it needs samtools). If use-cases outside of Galaxy don't require the packages then I strongly encourage you to not modify the bioconductor package, it's cleanest to then simply add more requirements to the Galaxy wrapper.

Reply:
OK thanks - I didn't release that about the mulled containers. 

I was originally using the suggest as per this description for bioconductor:

> **Suggests**: is for packages used in vignettes or examples, or in conditional code.

Where in my case I considered the parameter options that required the **suggested** packages to be in **conditional code**. 

However, now I think I should really move these packages to the "import" section of bioconductor as more functionality in the msPurity is becoming reliant on these "suggested" packages.

Reply:
That's fair, please make a PR updating the version and dependencies when/if you do that (the automatic PR generation skips bioconductor packages).

Reply:
But perhaps the best approach is that for this release of msPurity - I will do as you mention and add the additional packages to Galaxy.

And then for the next bioconductor release I will move the packages over to the "import" section.

So we can close this pull request for now

Reply:
:+1: Sounds like a good plan